pilot/pkg/config/kube/crdclient/client.go

```diff
@@ -517,7 +517,10 @@ func handleCRDAdd(cl *Client, name string, stop <-chan struct{}) {
 	resourceGVK := s.Resource().GroupVersionKind()
 	gvr := s.Resource().GroupVersionResource()
 
-	if cl.client.GetMemberRoll() != nil && resourceGVK == gvk.GatewayClass {
```

Comment:
I don't understand why we need this change?

Reply:
Right, it's enabled now :) Let me clean it up.

Reply:
It's a simple/naive merge from https://github.com/maistra/istio/pull/642/files

Reply:
I remember getting an error related to `GatewayClass` when I enabled analysis. That's why this change was neede. However, I think I may have found other issues afterwards that I found out were caused by some other issue. It's possible that after fixing this second issue, this change here is no longer needed. 

But I don't remember what that other issue/fix was.

Reply:
Ok, so I am going to merge as-is. But we should create an issue to follow up on this. Would it make sense @dgn @luksa?

Reply:
https://issues.redhat.com/browse/OSSM-2337